osd/incident_update.json

```diff
@@ -0,0 +1,10 @@
+{
+  "event_stream_id": "${INCIDENT_OR_JIRA_ID}",
+  "severity": "Info",
+  "service_name": "SREManualAction",
+  "log_type": "customer-support",
+  "summary": "Cluster incident update",
+  "description": "${INCIDENT_UPDATE_MESSAGE}",
```

Comment:
```suggestion
  "description": "${INCIDENT_UPDATE_MESSAGE}.",
```

Reply:
I'm also down to add a canned sentence here to "Refer to support case ${CASE} for direct communication." except worded better

Reply:
For completely open-ended SLs, I think I like not having the comma. It feels weird when typing it to do all the other parts of the sentence and then not type the period. 

But I do agree with a "Refer to" kind of statement. I'll run through some options.

Reply:
Updated

Reply:
I get it - it's also that the CI check in this repo enforces the message ends in a period haha (and this way senders can't forget one)